tests/main/writable-areas/task.yaml

```diff
@@ -1,5 +1,9 @@
 summary: Check that snap apps and services can write to writable areas.
 
+environment:
+    SNAP_REEXEC/norexec: 0
+    SNAP_REEXEC/withreexec: 1
+
```

Comment:
Can we please have a smoke test that ensures that hello world (or similar) works under reexec.